packages/xsnap/lib/console-shim.js

```diff
@@ -1,16 +1,13 @@
 /* global globalThis */
+
+// We use setQuote() below to break the cycle
+// where SES requires console and console is
+// implemented using assert.quote from SES.
+let quote = _v => '[?]';
+
 function tryPrint(...args) {
```

Comment:
There's no longer a `try/catch` here, so another name would be better.
For newly written functions (or even mostly newly written), please don't use non-arrow functions unless there's a reason.

```suggestion
const printAll = (...args) =>
  print(...args.map(v => typeof v === 'string' ? v : quote(v)));
```